litrpc/lit-sessions.proto

```diff
@@ -15,11 +15,12 @@ service Sessions {
 }
 
 enum SessionType {
+    reserved 4;
     TYPE_MACAROON_READONLY = 0;
     TYPE_MACAROON_ADMIN = 1;
     TYPE_MACAROON_CUSTOM = 2;
     TYPE_UI_PASSWORD = 3;
-    TYPE_MACAROON_ACCOUNT = 4;
+    TYPE_MACAROON_ACCOUNT = 5;
```

Comment:
I think this might also require a re-build of the JS stuff?

Reply:
aaahhh good catch!! updated!